src/app/ErrorView/ErrorView.tsx

```diff
@@ -44,15 +44,31 @@ export interface ErrorViewProps {
 }
 
 export const ErrorView: React.FunctionComponent<ErrorViewProps> = (props) => {
-    return (<>
-        <br/>
-        <Bullseye>
-            <ExclamationCircleIcon size='md' color='Red'/>
-        </Bullseye>
-        <Bullseye>
-            <Text>
+
+    if (props.message === 'Target SSL Untrusted'){
+        return (<>
+            <br/>
+            <Bullseye>
+                <ExclamationCircleIcon size='md' color='Red'/>
+            </Bullseye>
+            <Bullseye>
+                <Text>Error:&nbsp;{props.message}</Text>
+            </Bullseye>
+            <Bullseye>
+                <Text>
+                    To add the SSL certificate for this target, click <a href='/Security'>here</a> or go to the Security tab
```

Comment:
Using an `<a>` anchor like this causes a page refresh. I think the better mechanism to use here is what the "Create" button on the Active Recording List does, which gives in-application navigation with URL updates without refreshing the client.

Reply:
Also, I think associating the page movement with a button rather than an anchor link would fit in better with the style of the application. It could simply be a label like `To add the SSL certificate for this target, go to [Security]`, where `[]` denotes a clickable button.